ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/capabilities/baselinecaps/vset/provision/cucushift-installer-rehearse-azure-ipi-capabilities-baselinecaps-vset-provision-chain.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/azure/ipi/capabilities/baselinecaps/vset/provision/cucushift-installer-rehearse-azure-ipi-capabilities-baselinecaps-vset-provision-chain.yaml",
```

Comment:
Suggest to rename `capabilities` to `capability` to keep consistent style with `gcp/ipi/capability` folder

Reply:
updated